corona-warn-app/cwa-app-android
# Add Crowd Notifier Public Key for different environments (EXPOSUREAPP-6118)

Adds the crowd notifier public key to our environment setup and stores it in the `cnPublicKey` field of the `TraceLocation` database table after creating an event. 

See [TechSpec](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/event-registration-mvp/docs/spec/event-registration-client.md#crowd-notifier-public-key)

How to test: 
- update `test_environments.json` (see wiki) so that it includes `CROWD_NOTIFIER_PUBLIC_KEY`
- Create trace location via real organizer flow
- TestMenu -> EventReg -> Show Stored Events
- Check if created trace location contains proper `cnPublicKey`
